src/components/badges/BadgeBlock.tsx

```diff
@@ -1,70 +1,54 @@
-import { GoerliContractsStore } from 'stores/ContractsStore'
+import { BadgesContractsStore } from 'stores/ContractsStore'
 import { useSnapshot } from 'valtio'
 import { useState } from 'preact/hooks'
 import BadgeCard from 'components/badges/BadgeCard'
 import BadgeTitle from 'components/badges/BadgeTitle'
 import BadgeWrapper from 'components/badges/BadgeWrapper'
-import BaseProof from 'helpers/BaseProof'
-import Button from 'components/Button'
-import EmailBadge from 'icons/EmailBadge'
-import EmailProof from 'helpers/EmailProof'
-import Erc721Badge from 'icons/Erc721Badge'
+import Button from 'components/ui/Button'
 import MintedToken from 'models/MintedToken'
+import Proof from 'models/Proof'
 import ProofStore from 'stores/ProofStore'
 import WalletStore from 'stores/WalletStore'
+import data from 'data'
 import handleError from 'helpers/handleError'
 
 function Badge({
   proof,
   onMinted,
   onMintFailed,
 }: {
-  proof: BaseProof
+  proof: Proof
```

Comment:
Extract it into `inerface`, you have same thing [here](https://github.com/BigWhaleLabs/seal-cred-frontend/pull/452/files#diff-5b1516035d380825740282f22b8ac9801995a54f0ccbba08ab46bd23de3c8bdbR73)

Reply:
We still need to think about this, perhaps this interface is not suitable for new proofs